platform/solr/solr-factory/pom.xml

```diff
@@ -114,22 +294,22 @@
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.11</minimum>
+                                            <minimum>0.09</minimum>
```

Comment:
Those numbers are really low. Any chance we can add more test to `TestConfigurationFileProxy` to improve those?

If there's a good reason for them to be low, I would at least add a comment explaining why.